pgconfeu-recap.mdx

```diff
@@ -0,0 +1,180 @@
+slug: pgconfeu-recap-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d575fqbbtbk8h135drrsm2u27lu88
+---
+
+## PGConf.EU 2024 Recap
+
+We finished yet another PGConf.EU in Athens, Greece at the end of October.
+PGConf.EU is a very special conference for me, and I've written a blog post about it previously [here](https://xata.io/blog/pgconfeu-2024).
+
+Now that the conference is over, I'd like to share some more details about the conference, my experiences, and the talks I attended.
+
+## CloudNativePG Training
+
+Day O of PGConf.EU is when we do training sessions.
+My dear friends and former colleagues, Gabriele Bartolini and Leonardo Cecchi, held a [CloudNativePG training session](https://www.postgresql.eu/events/pgconfeu2024/schedule/session/5998-mastering-postgresql-in-kubernetes-with-cloudnativepg/).
+The training was about managing PostgreSQL databases within Kubernetes environments using the CloudNativePG operator.
+
+I've just stopped by to say hi to Gabriele and Leonardo and took a few photos and my colleague Andrew attended the training.
+We like CloudNativePG a lot at Xata, and if you are interested in learning more about it, I recommend checking out Gabriele's [blogs](https://www.gabrielebartolini.it/articles/) about it.
+And if you are using CloudNativePG, do not forget to give a star to the [CloudNativePG GitHub repository](https://github.com/cloudnative-pg/cloudnative-pg)!
+
+## PostgreSQL Europe Diversity Committee Meeting
+
+We had the first meeting of the PostgreSQL Europe Diversity Committee. Members of the committee (in alphabetical order) are:
+
+- Flavio Henrique Araque Gurgel
+- Floor Drees
+- Gülçin Yıldırım Jelínek
+- Jimmy Angelakos
+- Karen Jex
+- Stacey Haysler
+- Stefanie Janine Stölting
+- Valeria Kaplan
+
+We had a great time discussing the goals and plans for the Diversity Committee.
+
+The goals of the Diversity Committee are:
+
+- Broaden the diversity within the “active core” of the PostgreSQL community.
+- Enhance the visibility of underrepresented groups within the PostgreSQL community.
+- Raise awareness about the challenges faced by underrepresented groups within the community and highlight the value that diversity brings to it.
+- Address the “leaky pipeline” issue by lowering the entry barrier, improving the retention of newcomers, and increasing the engagement of the underrepresented individuals who are already part of the community.
+
+If you are interested in joining the Diversity Task Force, send us an email at diversity@postgresql.eu.
+
+## PostgreSQL Extension Ecosystem Summit & pgzx Talk
+
+After the PostgreSQL Europe Diversity Committee meeting, I've attended PostgreSQL Extension Ecosystem Summit, and gave talk about [pgzx](https://github.com/xataio/pgzx).
+
+Zig is a small and simple language that aims to be a "modern C" and make system-level code bases easier to maintain. It provides safe memory management, compilation time code execution (comptime), and a rich standard library.
```

Comment:
Maybe I'm just a fan of linking to things? lol

```suggestion
[Zig](https://ziglang.org/) is a small and simple language that aims to be a "modern C" and make system-level code bases easier to maintain. It provides safe memory management, compilation time code execution (comptime), and a rich standard library.
```

Reply:
Hehe, referencing is never a bad idea!